gitlab_to_stash.py

```diff
@@ -136,11 +136,14 @@ def main(argv=None):
     skipped_count = 0
     print('Processing GitLab projects...', file=sys.stderr)
     sys.stderr.flush()
-    for project in gen_all_results(git.getallprojects,
+    for project in gen_all_results(git.getprojects,
```

Comment:
If you do `git.getprojects` instead of `git.getallprojects`, you will miss everyone's private repositories (even if you're authenticated as an admin).